src/cli/sql.rs

```diff
@@ -13,8 +13,9 @@ use surrealdb::Response;
 
 #[tokio::main]
 pub async fn init(matches: &clap::ArgMatches) -> Result<(), Error> {
-	// Set the default logging level
-	crate::cli::log::init(0);
+	// Initialize o11y
+	crate::o11y::builder().with_log_level("warn").init();
+
```

Comment:
For the `sql` shell we had no errors being output on the client side. What are your thoughts here?

Reply:
what do you mean? you suspect there's a misconfiguration somewhere that hides the errors? I can try to reproduce